bot/exts/moderation/stream.py

```diff
@@ -70,6 +70,27 @@ async def _reload_tasks_from_redis(self) -> None:
                 self._revoke_streaming_permission(member)
             )
 
+    async def _suspend_stream(self, ctx: commands.Context, member: discord.Member) -> None:
+        """Suspend a member's stream."""
+        voice_state = member.voice
+
+        if not voice_state:
+            return
+
+        # If the user is streaming.
+        if voice_state.self_stream:
+            # End user's stream by moving them to AFK voice channel and back.
+            original_vc = voice_state.channel
+            await member.move_to(self.bot.get_channel(Channels.afk_voice))
+            await member.move_to(original_vc)
```

Comment:
You can get None from `get_channel` if the cache isn't ready yet. Usually this problem is circumvented by awaiting the `wait_until_guild_available` event on the bot instance. Although I guess it's not much of a concern here since it's only called from a command.

From the docs, it looks like if you do get None here it will kick the user from voice, and L85 will fail since you cannot reconnect them.

Reply:
Yea, we don't want to kick them from voice with this, so adding a wait until guild available makes sense.

Reply:
I think `member.voice` also uses the cache, hence I will wait for the cache ready even before I call `member.voice`